core/src/main/kotlin/ch/ergon/dope/resolvable/expression/unaliased/type/numberfunction/ArcCosineExpression.kt

```diff
@@ -0,0 +1,10 @@
+package ch.ergon.dope.resolvable.expression.unaliased.type.numberfunction
+
+import ch.ergon.dope.resolvable.expression.TypeExpression
+import ch.ergon.dope.resolvable.expression.unaliased.type.toDopeType
+import ch.ergon.dope.validtype.NumberType
+
+class ArcCosineExpression(value: TypeExpression<NumberType>) : NumberFunctionExpression("ACOS", value)
+
+fun acos(value: TypeExpression<NumberType>) = ArcCosineExpression(value)
+fun acos(value: Number) = acos(value.toDopeType())
```

Comment:
Same here and everywhere else. But this is for @jansigi to decide, as it concerns the coding style.